### ISSUE TITLE
Download Button Spacing on Mobile Screen

### ISSUE THREAD

**Issue by dorian-edwards** [quinwoods/catalystchildcare.org#18]
On mobile screen, there is no spacing between download button and footer.

![sws2](https://user-images.githubusercontent.com/47185542/175776462-68a49796-14b7-49a5-a923-74c303d1debd.PNG)

**Comment by quinwoods** [comment]
Was literally just thinking this as I was review your last PR!

**Comment by dorian-edwards** [comment]
Yeah, I should've did more testing before I submitted my PR. I'll sort it out as soon as I get home unless someone beats me to it
